addons/website_slides_survey/models/survey_user.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class SurveyUserInput(models.Model):
```

Comment:
I would add a small explanation about having slide_partner_id (member) or slide_id (test entry for publishers, should not happen in other cases).

Reply:
Added some "help" parameters on those two.